x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/list_alert_types.ts

```diff
@@ -65,7 +65,7 @@ export default function listAlertTypes({ getService }: FtrProviderContext) {
       describe(scenario.id, () => {
         it('should return 200 with list of globally available alert types', async () => {
           const response = await supertestWithoutAuth
-            .get(`${getUrlPrefix(space.id)}/api/alerts/list_alert_types`)
+            .get(`${getUrlPrefix(space.id)}/api/alerting/rule_types`)
```

Comment:
nit: rename this file to `rule_types` to reflect the route name change?

Reply:
Fixed in https://github.com/elastic/kibana/pull/95159/commits/f53a57a71d8a19877e7a9f7baa40cadc911ccbb3. We'll definitely need to do another sweep at these tests later on to refactor variable names, file names, test names, etc.